### PR TITLE
Homeslides

### PR DESCRIPTION
**Pull request by petealbertson**
This includes fixes for home slide animations, icons, and some text alignment/wrapping.

### CODE REVIEW

**Comment by zeekay** [comment]
Deploy preview ready!

Built with commit a35df270d9ec6ad4b43cd545e8e0a9376543d1ba

http://deploy-preview-57.ludela.netlify.com

File: src/css/components/visuals/intro.styl [hunk 1]
```diff
@@ -95,7 +95,7 @@ $phone_height = 390
   / .splash.is-step-3 &
     opacity 1
     // transition opacity 500ms ease
-    transition opacity, transform
+    transition-property opacity, transform
```
Comment (zeekay) [inline src/css/components/visuals/intro.styl]: You should use `transition` not `transition-property`. Autoprefixer is not able to correctly combine `transition-property` with any other `transition-*` properties: https://github.com/postcss/autoprefixer/blob/master/lib/transition.coffee#L73.

**Comment by zeekay** [comment]
Does swapping `transition-property` with `transition` break the animation sequence? It seems to be preferred by autoprefixer.

**Comment by petealbertson** [comment]
@zeekay yes. autoprefixer doesn't like `transition-property` but we're using it to specify delays times.

**Pull request merged**